plugin/loadbalance/README.md

```diff
@@ -19,7 +19,28 @@ implementations (like glibc) are particular about that.
 loadbalance [POLICY]
 ~~~
 
-* **POLICY** is how to balance. The default, and only option, is "round_robin".
+* **POLICY** is how to balance. Available options are `round_robin` and `weighted`. The default is `round_robin`.
+
+ The `round_robin` strategy randomizes the order of  A, AAAA, and MX records applying a uniform probability distribution.
+
+ The `weighted` policy assigns weight values to IPs to control the relative likelihood of particular IPs to be returned as the first
+(top) A/AAAA record in the answer. Note that it does not shuffle all the records in the answer, it is only concerned about the first A/AAAA record
+returned in the answer.
+
+
+~~~
+loadbalance weighted_round_robin WEIGHTFILE
+~~~
```

Comment:
Please remove this and consolidate it into with the first syntax definition (immediately under **Syntax** title).

Reply:
Done.